### ISSUE TITLE
Unable to  compile

### ISSUE THREAD

**Issue by JoePython1** [phildawes/racer#475]
Compiler errors noticed on Ubuntu 14.04

````shell
$ rustc --version
rustc 1.5.0 (3d7cd77e4 2015-12-04)


..............
  Compiling racer v1.1.0
.cargo/registry/src/github.com-0a35038f75765ae4/racer-1.1.0/src/racer/ast.rs:44:17: 44:27 error: mismatched types:
 expected `syntex_syntax::errors::DiagnosticBuilder<'_>`,
    found `syntex_syntax::errors::FatalError`
(expected struct `syntex_syntax::errors::DiagnosticBuilder`,
    found struct `syntex_syntax::errors::FatalError`) [E0308]
.cargo/registry/src/github.com-0a35038f75765ae4/racer-1.1.0/src/racer/ast.rs:44             Err(FatalError) => None
                                                                                                ^~~~~~~~~~
.cargo/registry/src/github.com-0a35038f75765ae4/racer-1.1.0/src/racer/ast.rs:44:17: 44:27 help: run `rustc --explain E0308` to see a detailed explanation
.cargo/registry/src/github.com-0a35038f75765ae4/racer-1.1.0/src/racer/ast.rs:56:17: 56:27 error: mismatched types:
 expected `syntex_syntax::errors::DiagnosticBuilder<'_>`,
    found `syntex_syntax::errors::FatalError`
(expected struct `syntex_syntax::errors::DiagnosticBuilder`,
    found struct `syntex_syntax::errors::FatalError`) [E0308]
.cargo/registry/src/github.com-0a35038f75765ae4/racer-1.1.0/src/racer/ast.rs:56             Err(FatalError) => {
                                                                                                ^~~~~~~~~~
.cargo/registry/src/github.com-0a35038f75765ae4/racer-1.1.0/src/racer/ast.rs:56:17: 56:27 help: run `rustc --explain E0308` to see a detailed explanation
.cargo/registry/src/github.com-0a35038f75765ae4/racer-1.1.0/src/racer/snippets.rs:61:21: 61:31 error: mismatched types:
 expected `syntex_syntax::errors::DiagnosticBuilder<'_>`,
    found `syntex_syntax::errors::FatalError`
(expected struct `syntex_syntax::errors::DiagnosticBuilder`,
    found struct `syntex_syntax::errors::FatalError`) [E0308]
.cargo/registry/src/github.com-0a35038f75765ae4/racer-1.1.0/src/racer/snippets.rs:61                 Err(FatalError) => {
                                                                                                         ^~~~~~~~~~
.cargo/registry/src/github.com-0a35038f75765ae4/racer-1.1.0/src/racer/snippets.rs:61:21: 61:31 help: run `rustc --explain E0308` to see a detailed explanation
error: aborting due to 3 previous errors
failed to compile `racer v1.1.0`, intermediate artifacts can be found at `/home/joepython/target-install`

Caused by:
  Could not compile `racer`.

To learn more, run the command again with --verbose.

**Comment by JoePython1** [comment]
The above error message happens when i do a cargo install. I am able to compile racer after doing a git clone on this repo though.
Thank you

**Comment by gsathya** [comment]
I just hit this on OSX El Capitan as well. 

```
> $ rustc --version
rustc 1.5.0 (3d7cd77e4 2015-12-04)
```

**Comment by phildawes** [comment]
I've released a new version 1.2.0 to crates.io. cargo install should work now (hopefully)
